app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -575,6 +575,12 @@ public void onDestroyView() {
         super.onDestroyView();
     }
 
+    public void setImageBlockingEnabled(boolean enabled) {
+        for (Tab tab : tabsSession.getTabs()) {
+            tab.setImageBlockingEnabled(enabled);
+        }
+    }
+
```

Comment:
it would be sweet if we add `TODO: this implementation should not belong to Fragment` to this method, as well as `setContentBlockingEnabled`

Reply:
comment added